### ISSUE TITLE
Tarnet short tutorial error

### ISSUE THREAD

**Issue by xiaogangzhu** [kochbj/Deep-Learning-for-Causal-Inference#7]
Hi,

When i am running first tutorial of TarNet on colad, got the training loss = nan.
![Screenshot from 2023-05-15 14-17-32](https://github.com/kochbj/Deep-Learning-for-Causal-Inference/assets/65101307/f33a0375-8879-4281-99ef-1dbd80bbb1cf)

**Comment by kochbj** [comment]
Hi,

I fixed the TF version at 2.8. Hopefully that solves the issue for you. Please let me know if you find other bugs!

Warmly,
Bernie
